packages/@lwc/engine-core/src/framework/api.ts

```diff
@@ -205,30 +207,28 @@ function s(
                         );
                     }
                     // Ignore slot content from parent
-                    return accumulator;
+                    continue;
                 }
                 // If the passed slot content is factory, evaluate it and add the produced vnodes
                 if (assignedNodeIsScopedSlot) {
                     const vmBeingRenderedInception = getVMBeingRendered();
-                    let scopedSlotChildren: VNodes = [];
                     // Evaluate in the scope of the slot content's owner
                     // if a slotset is provided, there will always be an owner. The only case where owner is
                     // undefined is for root components, but root components cannot accept slotted content
                     setVMBeingRendered(slotset.owner!);
                     try {
-                        scopedSlotChildren = vnode.factory(data.slotData);
+                        ArrayPush.apply(newChildren, vnode.factory(data.slotData) as VNode[]);
```

Comment:
I considered isolating just the `factory()` call here for readability, but it adds quite a bit of extra code:

```ts
const vmBeingRenderedInception = getVMBeingRendered();
let scopedSlotChildren: VNodes | undefined;
setVMBeingRendered(slotset.owner!);
try {
    scopedSlotChildren = vnode.factory(data.slotData);
} finally {
    setVMBeingRendered(vmBeingRenderedInception);
}
if (!isUndefined(scopedSlotChildren)) {
    ArrayPush.apply(newChildren, scopedSlotChildren as VNode[]);
}
```

Reply:
The return value from the factory need to be spread out and not inserted as-is.

Reply:
@ravijayaramappa This is `Array.push.apply`, not `ArrayPush.call`. So it is being spread out. 🙂

Reply:
🤦 oh yes, my bad.